File: sdkwork-app-sdk-python/sdkwork_app_sdk/api/bind.py
from typing import Any, Dict, List, Optional
from ..http_client import HttpClient
from ..models import PlusApiResultVoid, ThirdPartyBindForm

class BindApi:
    """bind API client."""
    
    def __init__(self, client: HttpClient):
        self._client = client

    def third_party_account(self, platform: str, body: ThirdPartyBindForm) -> PlusApiResultVoid:
        """绑定第三方账号"""
        return self._client.post(f"/app/v3/api/user/bind/{platform}", json=body)

    def unbind_third_party_account(self, platform: str) -> PlusApiResultVoid:
        """解绑第三方账号"""
        return self._client.delete(f"/app/v3/api/user/bind/{platform}")

File: sdkwork-app-sdk-flutter/lib/src/api/bind.dart
import '../http/client.dart';
import '../models.dart';

class BindApi {
  final HttpClient _client;
  
  BindApi(this._client);

  /// 绑定第三方账号
  Future<PlusApiResultVoid?> thirdPartyAccount(String platform, ThirdPartyBindForm body) async {
    final response = await _client.post(ApiPaths.appPath('/user/bind/${platform}'), body: body, contentType: 'application/json');
    return response is PlusApiResultVoid ? response : null;
  }

  /// 解绑第三方账号
  Future<PlusApiResultVoid?> unbindThirdPartyAccount(String platform) async {
    final response = await _client.delete(ApiPaths.appPath('/user/bind/${platform}'));
    return response is PlusApiResultVoid ? response : null;
  }
}
